### PR TITLE
Rata/userns internal review

### PR DESCRIPTION
**Pull request by rata**
cc @giuseppe

### CODE REVIEW

File: keps/sig-node/127-user-namespaces/README.md [hunk 1]
```diff
@@ -270,15 +275,20 @@ we don't want to forget or want to highlight it. Some things that are obvious we
 need to tackle are not listed. Let us know if you think it is important to add
 something else to this list:
 
-- We will start with mappings of 64K. Tim Hockin, however, has expressed
+- We will start with mappings of 64K. Tim Hockin, however, have expressed
```
Comment (giuseppe) [inline keps/sig-node/127-user-namespaces/README.md:280]: should this still be `has`?

**Review by giuseppe — COMMENTED** [review]
LGTM, just a nit

File: keps/sig-node/127-user-namespaces/README.md (hunk 1 above)
Comment (rata) [reply keps/sig-node/127-user-namespaces/README.md:280]: ouch, yes. Will fix that and update the upstream PR. Thanks!

**Comment by rata** [comment]
@giuseppe updated with the CRI changes. PTAL

File: keps/sig-node/127-user-namespaces/README.md [hunk 2]
```diff
@@ -200,6 +200,65 @@ If true or not present, uses the host user namespace (as today)
...
+The following messages will be added:
+
+```
+// A IDMapping describes a ID mapping for the user namespace used for the
+// pod.
+message IDMapping {
+    // host_id is the id on the host.
+    uint32 host_id = 1;
+    // host_id is the id in the container.
+    uint32 container_id = 2;
```
Comment (giuseppe) [inline keps/sig-node/127-user-namespaces/README.md:214]: the doc is wrong.  Have I done the same mistake in the api.proto file? :-)

**Pull request closed without merging**